### ISSUE TITLE
Cannot delete SQS queue with '/' in queue-name

### ISSUE THREAD

**Issue by hertzsprung** [localstack/localstack#3180]
# Type of request: This is a ...

[x] bug report
[ ] feature request

# Detailed description

localstack can create but cannot delete queues with '/' in their name.

## Expected behavior

```
Done.
```

## Actual behavior

```
An error occurred (AWS.SimpleQueueService.NonExistentQueue) when calling the DeleteQueue operation: The specified queue does not exist for this wsdl version.
```

# Steps to reproduce

## Command used to start LocalStack

```
START_WEB=0 localstack run
```

## Client code (AWS SDK code snippet, or sequence of "awslocal" commands)
```
aws --endpoint-url=http://localhost:4566 sqs create-queue --queue-name 123/abc
aws --endpoint-url=http://localhost:4566 sqs list-queues
aws --endpoint-url=http://localhost:4566 sqs delete-queue --queue-url http://localhost:4566/000000000000/123/abc
```

**Comment by whummer** [comment]
Thanks for reporting @hertzsprung . This should be fixed in #3192 - can you please pull the `latest` Docker image and give it another try? Please report here if the problem persists. Thanks

**Comment by hertzsprung** [comment]
Confirmed as fixed.  Thanks for the quick turnaround!
